userspace/libscap/engine/savefile/converter/results.h

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2024 The Falco Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+#pragma once
+
+typedef enum conversion_result {
```

Comment:
I'd move this type to `types.h` to avoid headers pollution.

Reply:
`conversion_result` is a public enum so we need to expose it to consumers if I move it into `types.h` we will end up exposing all the types. That's why I created a separate `.h`, do you have other ideas?

Reply:
ah i see, thanks that makes sense actually.